cvat-core/src/annotations-collection.ts

```diff
@@ -1026,6 +1026,7 @@ export default class Collection {
                         label_id: state.label.id,
                         outside: state.outside || false,
                         occluded: state.occluded || false,
+                        hidden: state.hidden || false,
```

Comment:
I am not sure that creating a new object already hidden makes sense as it confuses the user 
They press N and expect a new object to be appear, I think.
If the object is hidden, they may think that something is going wrong and start drawing it again  (especially when there are a lot of objects in the sidebar, newly added is not visible in the list)

Reply:
Hm, maybe it is for new objects. 
But what about edited ones? Should this hide feature affect state.hidden at all then?